lstchain/calib/camera/calibration_calculator.py

```diff
@@ -206,11 +206,8 @@ def process_interleaved(self, event):
         new_ped = False
         new_ff = False
 
-        # if pedestal event (check also the ucts trigger for the moment)
-        if (
-                LSTEventType.is_pedestal(event.r1.tel[self.tel_id].trigger_type) or
-                LSTEventType.is_pedestal(event.lst.tel[self.tel_id].evt.ucts_trigger_type)
```

Comment:
To understand this: the ucts_trigger_type wrongly identifies an event as a pedestal although it is not?

How does that happen?

Reply:
there is a hardware problem that is being worked out. Until then, we cannot trust on this flag

Reply:
@DirkHoffman seems to disagree that this is a hardware problem

Reply:
There have been several problems of UCTS/TIB bit disappearance at EVB and/or event class misidentification and there are many open discussion in camera/analysis calls.
The bottomline is that we cannot trust this flag for the time being.